proj1.css

```diff
@@ -1,6 +1,6 @@
 body {
       text-align: center;
-      background: url("file:///Users/kohchengchuan/wdi-sg-playground/project1/image/background3.jpeg");
+      background: url("file:///Users/kohchengchuan/project1/image/background3.jpeg");
```

Comment:
Drop the `file:` URL. Use relative path instead so the browser will know where to look for the file whether it's loaded from a web server or your local file system.